project/src/views/new-evaluation/technicalAreaPicker.js

```diff
@@ -0,0 +1,14 @@
+const Component = function (options = {}) {
+    const arr = options.map(el => `<div class="content-box-fields">
```

Comment:
putting the content after ` on a new line will allow you to have more room here and will look cleaner